### PR TITLE
Button name and value props example & CheckboxWrapper reset fix only in chrome (Issue/2642-2646-follow-up)

### PR DESCRIPTION
**Pull request by henrijoss**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2642-2646-follow-up

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves 
[#2646](https://github.com/porsche-design-system/porsche-design-system/issues/2646)
[#2642](https://github.com/porsche-design-system/porsche-design-system/issues/2642)

### CODE REVIEW

File: packages/components/src/components/button/button.examples.md [hunk 1]
```diff
@@ -82,13 +82,33 @@ By setting the `tabindex` attribute to `-1` you can remove the **Button** from t
...
+    <p-button name="option" value="B" type="submit">Button B</p-button>
+  </form>
+  <p-text :theme="theme">{{ selectedValuesForm }}</p-text>
+</Playground>
+
 <script lang="ts">
 import Vue from 'vue';
-import Component from 'vue-class-component';
+import Component from 'vue-class-component'; 
+import {getButtonCodeSamples} from "shared/src";
```
Comment (denyo) [inline packages/components/src/components/button/button.examples.md:100]: Please import from the built package.

File: packages/components/src/components/button/button.examples.md [hunk 2]
```diff
@@ -82,13 +82,33 @@ By setting the `tabindex` attribute to `-1` you can remove the **Button** from t
...
 <Playground :markup="taborder" :config="config"></Playground>
 
+## Form
+
+When used as a submit button, the `name` and `value` props are submitted as a pair as part of the form data.
+
+<Playground :frameworkMarkup="formExample" :config="{ ...config, withoutDemo: true }">
+  <form @submit.prevent="onSubmit">
+    <p-button name="option" value="A" type="submit" style="margin-right: 16px;">Button A</p-button>
+    <p-button name="option" value="B" type="submit">Button B</p-button>
```
Comment (denyo) [inline packages/components/src/components/button/button.examples.md:92]: Isn't `theme` missing here?

File: packages/components/src/utils/supportsChromiumMediaQuery.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { JssStyle } from 'jss';
```
Comment (denyo) [inline packages/components/src/utils/supportsChromiumMediaQuery.ts:1]:
```suggestion
import type { JssStyle } from 'jss';
```

File: packages/components/src/styles/checkbox-radio-styles.ts [hunk 4]
```diff
@@ -117,21 +117,24 @@ export const getCheckboxRadioJssStyle = (
...
-                transition: 'unset', // Fixes chrome bug where border-color is stuck on hover color
                 borderColor: checkedHoverColor,
                 backgroundColor: checkedHoverColor,
                 ...prefersColorSchemeDarkMediaQuery(theme, {
                   borderColor: checkedHoverColorDark,
                   backgroundColor: checkedHoverColorDark,
                 }),
               },
+              '.text:hover ~ &(input)': {
+                ...supportsChromiumMediaQuery({
```
Comment (denyo) [inline packages/components/src/styles/checkbox-radio-styles.ts:134]: Useless object composition.

File: packages/components/src/styles/checkbox-radio-styles.ts [hunk 5]
```diff
@@ -117,21 +117,24 @@ export const getCheckboxRadioJssStyle = (
...
+                  transition: 'unset', // Fixes chrome bug where border-color is stuck on hover color
```
Comment (denyo) [inline packages/components/src/styles/checkbox-radio-styles.ts:135]: Wouldn't resetting just the `border-color` transition more precise?
And I guess this selector is now strong enough to handle both cases, `input` and `input:checked`?

File: packages/components/src/utils/supportsChromiumMediaQuery.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+/**
+ * Applies a style only on Chromium based browsers by using a media query which is only supported there.
+ *
+ * @param {JssStyle} style - The style to be applied when the Chromium media query is supported.
+ * @returns {JssStyle} - The Chromium media query containing the style.
+ */
+export const supportsChromiumMediaQuery = (style: JssStyle): JssStyle => ({
+  '@media screen and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: 0.001dpcm)': style,
```
Comment (denyo) [inline packages/components/src/utils/supportsChromiumMediaQuery.ts:10]: Would be good to document a link to the source where this is coming from.

File: packages/components/src/styles/checkbox-radio-styles.ts (hunk 5 above)
Comment (henrijoss) [reply packages/components/src/styles/checkbox-radio-styles.ts]: All transition properties can get stuck, i updated the comment. Selector is strong enough for both.

**Review by denyo — APPROVED** [review]

**Review by denyo — APPROVED** [review]

**Pull request merged**